tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -24,16 +24,16 @@ def test_it(self, lti_launch, course_service):
 
 class TestResourceLinkIdk:
     @pytest.mark.parametrize(
-        "learner_id,get_id,post_id,expected",
+        "learner_id,get_id,lti_id,expected",
         [
-            param(None, None, "POST_ID", "POST_ID", id="regular"),
-            param("USER_ID", "GET_ID", "POST_ID", "GET_ID", id="new_speedgrader"),
-            param("USER_ID", None, "POST_ID", "POST_ID", id="old_speedgrader"),
+            param(None, None, "LTI_ID", "LTI_ID", id="regular"),
+            param("USER_ID", "GET_ID", "LTI_ID", "GET_ID", id="new_speedgrader"),
+            param("USER_ID", None, "LTI_ID", "LTI_ID", id="old_speedgrader"),
```

Comment:
:+1: Good improvement to specificity here. Took me a couple of reads to try and work out why though!